src/functionalTest/java/uk/gov/hmcts/sptribs/caseworker/CaseworkerAddNoteFT.java

```diff
@@ -21,7 +21,11 @@
 public class CaseworkerAddNoteFT extends FunctionalTestSuite {
 
     private static final String REQUEST = "classpath:request/casedata/ccd-callback-casedata.json";
+    private static final String REQUEST_MULTIPLE_NOTES = "classpath:request/casedata/ccd-callback-casedata-multiple-notes.json";
```

Comment:
Small nitpick - can we rename these constants and response constants below to specify they are for about to submit. Also can you rename the json file to include the callback name (in this case about-to-submit)